libraries/AP_GPS/AP_GPS_UBLOX.cpp

```diff
@@ -628,6 +644,15 @@ AP_GPS_UBLOX::_parse_gps(void)
                     break;
                 }
             }
+        } else if(_msg_id == MSG_ACK_NACK) {
+            switch(_buffer.ack.clsID) {
+            case CLASS_CFG:
+                switch(_buffer.ack.msgID) {
+                case MSG_CFG_SBAS:
+                    _unconfigured_messages &= ~CONFIG_SBAS;
```

Comment:
because not all gps's have a valid sbas option. m8p is one of them. without this change the request loop stays at 750ms. vs the 4000ms